src/scripts/process_Institution.py

```diff
@@ -241,7 +241,6 @@ def __compare_two_rows_Inst(row, row_gh):
 
 def __overwrite_Inst(row, row_gh, index, today):
     fields_to_be_overwritten = [
-        "inst_name",
```

Comment:
Will empty `inst_name` fields still be populated, or is this a partial feature removal?

Reply:
> Will empty `inst_name` fields still be populated, or is this a partial feature removal?

Removal. As we saw that there could be multiple `inst_name` for the same Institution in the same language, therefore, I realized that to update the `inst_name` is not a good idea. Therefore, it is removed from the wait-to-be-updated fields.

Reply:
ok in that case we need to bump the version to `2.0.0`